app/routes/bdb/components/bdb.css

```diff
@@ -134,11 +134,11 @@
 }
 
 .contacted {
-  background-color: #ffff7f;
+  background-color: var(--color-white);
```

Comment:
Here as well

Reply:
Reverted, as this color should not be changed in darkMode